unfetter-discover/unfetter-ui
# Initial baselines wizard (Issue 927)

Fixes unfetter-discover/unfetter#927
Requires unfetter-discover/unfetter#939

This establishes the initial part of the assessments 3.0 baseline wizard, with the object assessment being created with its base settings and only the most basic of unit tests.  Further steps in the wizard, along with unit tests for those steps, are to follow.

## To test:
1. Pull this PR and unfetter-discover/unfetter#939
2. Restart stack
3. Navigate to Assessments 3.0
4. Verify a basic summary screen appears for an existing baseline (object assessment)
5. Click the down triangle next to baseline name on the left
6. On list of object assessments, click Create
7. Verify the baseline settings panel appears and enter title, description, and submitter org
8. Click Continue
9. Verify initial wizard appears, though nothing useful appears yet...
